Cargo.toml

```diff
@@ -14,12 +14,19 @@ categories = ["gui"]
 
 license = "MIT OR APACHE-2.0"
 
+[features]
+default = ["x11"]
+x11 = ["xcb"]
+
 [dependencies]
-winit = "0.23.0"
+winit = "0.24.0"
 
-[target.'cfg(windows)'.dependencies]
+[target.'cfg(target_os = "windows")'.dependencies]
 winapi = { version = "0.3.8", features = ["windef", "winuser"] }
 
+[target.'cfg(target_os = "linux")'.dependencies]
+xcb = { version = "0.9.0", features = ["composite"], optional = true }
```

Comment:
Is there a particular reason you chose XCB over Xlib?
Notably, the Rust XCB bindings don't support dynamically linking XCB, which is what I'd prefer to do **if** implementing this is possible with both Xlib and XCB.

Reply:
At first, I try to use Xlib. But I don't see any bindings around Shape Extension to use with Xlib in [the crate which winit uses](https://martindev.github.io/martock/doc/x11_dl/index.html) or any other crates.  
On the other hand, I can find the one to use with XCB. That's the reason I choose XCB.
I'm not familiar with X11, so there might be another way to implement this future using Xlib.

Reply:
Alright. Supporting both isn't something I'd consider necessary at the moment.
I wasn't really expecting to get X11 support in this crate any time soon, anyway, as my attention has been elsewhere.